examples/run_CPMG.py

```diff
@@ -13,15 +13,13 @@
 import os, sys
 from numpy import *
 import SpinCore_pp
-from SpinCore_pp import prog_plen, get_integer_sampling_intervals
+from SpinCore_pp import prog_plen, get_integer_sampling_intervals, save_data
 from SpinCore_pp.ppg import generic
 from datetime import datetime
 from Instruments.XEPR_eth import xepr
-import h5py
 
 my_exp_type = "ODNP_NMR_comp/CPMG"
-target_directory = getDATADIR(exp_type=my_exp_type)
```

Comment:
define the target directory here  still so that you can pass it to save_data later

Reply:
still not fixed